### PR TITLE
fix: Minting live in drop page not pulsing

### LINKED ISSUE #9353: Minting live in drop page not pulsing
image
it should pulse like the indicators on landing page 
image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Bugfix

  ## Needs Design check

  - @exezbcz please review

  ## Context

  - [x] Closes #9353

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI

Kapture 2024-02-13 at 11 51 25

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

**Comment by exezbcz** [comment]
yup, just without the stroke please
- it was a example how the pulsing animation look like :D

**Comment by Jarsen136** [comment]
> yup, just without the stroke please
> 
> * it was a example how the pulsing animation look like :D

Okay, thanks

**Comment by prury** [comment]
effect is so subtle on dark mode, almost can't see

**Comment by preschian** [comment]
@Jarsen136 Maybe we can use animation from Tailwind, if that match with the design https://tailwindcss.com/docs/animation#ping

**Comment by exezbcz** [comment]
@preschian this one is close enough yup, why not, maybe slightly slower?

**Comment by Jarsen136** [comment]
> @Jarsen136 Maybe we can use animation from Tailwind, if that match with the design https://tailwindcss.com/docs/animation#ping

It's a good idea. I have applied the icon with `k-primary` colour.

Kapture 2024-02-13 at 20 35 52


> @preschian this one is close enough yup, why not, maybe slightly slower?

@exezbcz It's because the animation on svg is unstable and it somehow does not work. With the animation icon from tailwind, the issue could be solved better. Does it match the design?

**Comment by preschian** [comment]
> @preschian this one is close enough yup, why not, maybe slightly slower?

yes, we can adjust the duration https://tailwindcss.com/docs/animation#customizing-your-theme

**Comment by Jarsen136** [comment]
> > @preschian this one is close enough yup, why not, maybe slightly slower?
> 
> yes, we can adjust the duration https://tailwindcss.com/docs/animation#customizing-your-theme

Now It would be slower now with duration from 1s to 2s.

**Review by preschian — APPROVED** [review]

File: components/collection/voteDrop/DropContainer.vue [hunk 1]
```diff
@@ -23,7 +23,12 @@
                 {{ $t('mint.unlockable.phase') }}
               </div>
               <span v-if="mintCountAvailable" class="flex items-center">
-                <img src="/unlockable-pulse.svg" alt="open" />
+                <span class="relative flex h-3 w-3 mr-2">
+                  <span
+                    class="animate-ping absolute inline-flex h-full w-full rounded-full bg-k-primary opacity-75"></span>
+                  <span
+                    class="relative inline-flex rounded-full h-3 w-3 bg-k-primary"></span>
+                </span>
```
Comment (roiLeo) [inline components/collection/voteDrop/DropContainer.vue:31]: make it as component if you are going to reuse it? `PulseBadge`?

File: components/drops/TimeTag.vue [hunk 2]
```diff
@@ -1,8 +1,15 @@
 <template>
   <div
     class="tag-container rounded-[2rem] flex border py-1 px-2 justify-between items-center">
-    <div v-if="showIcon" class="image is-24x24 text-center">
-      <img v-if="isMintingLive" src="/drop/unlockable-pulse.svg" />
+    <div
+      v-if="showIcon"
+      class="image is-24x24 text-center flex items-center justify-center">
```
Comment (roiLeo) [inline components/drops/TimeTag.vue:6]: Bulma

**Review by roiLeo — APPROVED** [review]
otherwise lgtm

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 7.66 USD/DOT ~ 3.916 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x3b1b344e6b8590143099d37183a2f9519872c30324ff81483bf8aac9d84ac69a](https://polkadot.subscan.io/extrinsic/0x3b1b344e6b8590143099d37183a2f9519872c30324ff81483bf8aac9d84ac69a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**